### ISSUE TITLE
SQLite binaries should be upgraded to latest patched sqlite 3.32.1, due critical and high vulnerabilities

### ISSUE THREAD

**Issue by redabc** [TryGhost/node-sqlite3#1358]
Hi Mapbox team,

Recently our company internal docker image scanner reported a bunch of critical and high vulnerabilities related to the sqlite binaries version 3.31.1 which is used by the sqlite3 npm package version 5.0.0.

Is there any planned activity to perform the upgrade of the latest sqlite distribution version 3.32.1 from 2020-05-25?

Looking forward your soon feedback.

Thank you in advance.

**Comment by n1ru4l** [comment]
@redabc According to https://github.com/mapbox/node-sqlite3/commit/6e250c64199ce47ea8357a0a333f63ae0133e664 the bundles version is 3.32.3, which means this issue can be closed?

**Comment by n1ru4l** [comment]
Also, duplicate of https://github.com/mapbox/node-sqlite3/issues/1340

**Comment by nstemmler** [comment]
Hm, I'm recently upgraded the version of sqlite3 I'm using in one of my projects from 4.2 to version 5.0.0. I was excited to see this upgrade since versions >= 3.32.0 raised the default upper bound on the number of bound parameters from 999 to 32766 and I could really use this functionality in my application. It seems that even with version 5.0.0 of sqlite3 installed however, the pre-bundled version of sqlite3 is actually the 01/27/2020 release, which is version 3.31.1. I verified this by running a SQLite query using this library with more than 999 bound parameters (how I discovered this) yields the familiar 'SQLITE_ERROR: too many SQL variables', inspecting the bundled node_sqlite3.node binary, and by looking at the version in common-sqlite.gypi (3310100).

I'm not sure if this is just a distribution issue. I'm running on Node.js version 12.18.2 so it might be that the prebuilt bundled version of node_sqlite3.node for my version is not the latest but I was hoping to get the latest. I guess I'm going to look into building this library myself with the latest version for the time being.
